internal/provider/resource_secret_store.go

```diff
@@ -92,6 +94,9 @@ func (*SecretStore) Schema(ctx context.Context, req resource.SchemaRequest, resp
 			"id": schema.StringAttribute{
 				MarkdownDescription: "The ID of the Secret Store.",
 				Required:            true,
+				PlanModifiers: []planmodifier.String{
```

Comment:
Can we add some integration tests for this?

Reply:
This isn't a modifier test per se, but I've added a test that changes the id and verifies that the new resource reflects the updated id

Reply:
What I'd like to see if a proof that the old secret store has been deleted and the new one has the new id. The 1st check might require a Humanitec client to fetch api.